drivers/unix/file_access_unix.cpp

```diff
@@ -87,6 +92,24 @@ Error FileAccessUnix::open_internal(const String &p_path, int p_mode_flags) {
 		}
 	}
 
+#if defined(TOOLS_ENABLED)
+	if (p_mode_flags & READ) {
+		String real_path = get_real_path();
+		if (real_path != path) {
+			if (real_path.to_lower() != path.to_lower()) {
+				// This is a symlink, some platform may not support them.
+				WARN_PRINT_ONCE_ED("Tried to open a symlink file '" + path + "', referencing to '" + real_path + "' in the filesystem. This file will not open when exported to platforms without symbolic links, like Windows versions before Windows 10.");
```

Comment:
I'm using `WARN_PRINT_ONCE_ED` because for some reasons, it real spam the console with `17` messages each time I save my script ><'

Do you think I should also change the `WARN_PRINT` to `WARN_PRINT_ONCE_ED` in the windows version of this check ?